.github/workflows/ci.yml

```
@@ -6,6 +6,9 @@ on:
   pull_request:
     types: [opened, labeled, synchronize]
 
+env:
+  NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
```

just adding an explanation of what's going on here (i had to have a quick think)

```suggestion
  # The `accessToken` in `nx.json` is a read-only token for use in dev.
  # This overrides it with a read-write token so that CI run results are 
  # remotely cached.
  NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
```